tests/helpers/skipif.py

```diff
@@ -21,52 +21,56 @@
 from pytorch_lightning.utilities import _TORCH_QUANTIZE_AVAILABLE
 
 
-def skipif_args(
-    min_gpus: int = 0,
-    min_torch: Optional[str] = None,
-    quant_available: bool = False,
-) -> dict:
-    """ Creating aggregated arguments for standard pytest skipif, sot the usecase is::
-
-        @pytest.mark.skipif(**create_skipif(min_torch="99"))
-        def test_any_func(...):
-            ...
+class LitSkipif:
+    """
+    Light SkipIf wrapper for simple marking specific cases, fully compatible with pytest.mark::
 
-    >>> from pprint import pprint
-    >>> pprint(skipif_args(min_torch="99", min_gpus=0))
-    {'condition': True, 'reason': 'Required: [torch>=99]'}
-    >>> pprint(skipif_args(min_torch="0.0", min_gpus=0))  # doctest: +NORMALIZE_WHITESPACE
-    {'condition': False, 'reason': 'Conditions satisfied, going ahead with the test.'}
+        @LitSkipif(min_torch="0.0")
+        @pytest.mark.parametrize("arg1", [1, 2.0])
+        def test_wrapper(arg1):
+            assert arg1 > 0.0
     """
-    conditions = []
-    reasons = []
 
-    if min_gpus:
-        conditions.append(torch.cuda.device_count() < min_gpus)
-        reasons.append(f"GPUs>={min_gpus}")
+    def __new__(
+        self,
+        *args,
+        min_gpus: int = 0,
+        min_torch: Optional[str] = None,
+        quantization_missing: bool = False,
```

Comment:
It doesn't make sense to add `_missing` suffix. Then, it would need to be extended to horovod, deepspeed, etc. I think a boolean with the term `quantization` is intuitive and enough.

Reply:
done